server/src/test/java/org/candlepin/model/ConsumerTest.java

```diff
@@ -484,7 +484,7 @@ public void testRoleConvertedToNull() throws Exception {
         consumerCurator.clear();
 
```

Comment:
We should rename the test. Behavior changed so the expectation roleConvertedToNull is no longer valid. Same with usage below.

Reply:
Yes. Renamed the test cases.